keras/losses.py

```diff
@@ -1468,11 +1469,14 @@ def _ragged_tensor_mae(y_true, y_pred):
               'keras.losses.mean_absolute_percentage_error',
               'keras.losses.mape', 'keras.losses.MAPE')
 @tf.__internal__.dispatch.add_dispatch_support
-def mean_absolute_percentage_error(y_true, y_pred):
+def mean_absolute_percentage_error(y_true, y_pred, ignore_zeros):
```

Comment:
Please make ignore_zeros a keyword arg so that it will have a default value (which should be False), otherwise it will break all the existing caller to this API.